tools.js

```diff
@@ -59,6 +59,9 @@ module.exports = (function () {
 		timid: {name:"Timid", plus:'spe', minus:'atk'}
 	};
 
+	var init = typeof global.Symbol === 'function' ? Symbol('init') : '_init';
```

Comment:
Making it a symbol prevents from directly modifying the initialization method in regular use when they should be using only ``init``; e.g. the following example won't cause unexpected behavior.

```
exports.BattleScripts = {
init: function () {
// do stuff
_this.init(foo);
_this.init(bar);
},
_init: function (baz) {
// do stuff
}
};
```

The existence of a second property copying it is intended to prevent is inheritance by making it non-enumerable. (Non-configurability might be a bit unneeded...). Directly modifying the ``'init'`` property to forcing its non-enumerability might cause unexpected results as well.

Therefore, I deem the definition of another property as a symbol as the best pathway. Note that if you realy really want to access a symbol from outside, you can call ``Object.getOwnPropertySymbols``.